infra/license/README.md

```diff
@@ -0,0 +1,54 @@
+# License Self Verification
+
+## Background
+
+Numerous kinds of Node packages are needed to build and use ONE-vscode but some packages may have unacceptable license for both ONE and ONE-vscode.
+Therefore we should review whether licenses of each package can be used or not.
+However, if we check all of them manually, we may mistake or miss something.
+To reduce such risk and labor, automatic license checker would be good solution.
+
+## Process & Architecture
+
+### Flowchart
+
+![flowchart](flowchart.png)
```

Comment:
With `License is UNKNOWN`, I misunderstood that `UNKNOWN` was a type of license since all chars were capital. 
Can we use lower case letters, `unknown`?

Reply:
Following example includes `UNKNOWN` license and it is result of `license-checker`.

```
  "traverse@0.3.9": {
    "licenses": "UNKNOWN",
    "repository": "https://github.com/substack/js-traverse",
    "publisher": "James Halliday",
    "dependencyPath": "/home/seok/HDD/repos/ONE-vscode/node_modules/traverse",
    "path": "/home/seok/HDD/repos/ONE-vscode/node_modules/traverse",
    "licenseFile": "/home/seok/HDD/repos/ONE-vscode/node_modules/traverse/LICENSE"
  }
```

I just copied this :)
If `unknown` is better for flowchart, I will use `unknown` :D

Reply:
I didn't know that there is a license type `"UKNOWN"`. 😮 
Also I confessed that I thought that "UNKNOWN" was same with "Unlicensed"... but actually they are different.
- `Unlicense` type: https://choosealicense.com/licenses/unlicense/

I am now fine with `UNKNOWN` with all capital letter.